### PR TITLE
[AIFBR 7489] Updated CV  metadata

### PR DESCRIPTION
**Pull request by SahilChaudhary25**

### CODE REVIEW

File: metadata/ComputerVision__1__metadata.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
   "projectId": "[project-id]",
   "retrainable": false,
   "stagingUri": "[staging-uri]",
-  "projectName": "UiPath Document Understanding",
-  "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+  "projectName": "UiPath Computer Vision",
+  "projectDescription": "Backend server for the Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
```
Comment (mohitseth-nitrr) [inline metadata/ComputerVision__1__metadata.json:18]: I am doubtful about Backend server bit ... Should we be mentioning in projectDescription ?
cc @vivekjhauipath

**Review by mohitseth-nitrr — CHANGES_REQUESTED** [review]
clarification needed

File: metadata/ComputerVision__1__metadata.json (hunk 1 above)
Comment (vivekjhauipath) [reply metadata/ComputerVision__1__metadata.json:18]: Metadata comes from individual teams/PMs.
@SahilChaudhary25 did we get this from Stefan/Cosmin?

Comment (SahilChaudhary25) [reply metadata/ComputerVision__1__metadata.json:18]: It is given by cosmin

**Review by rajivml — APPROVED** [review]
ok

**Review by adeepsri — APPROVED** [review]
LGTM

**Pull request merged**